backend/src/main/scala/cromwell/backend/standard/callcaching/StandardCacheHitCopyingActor.scala

```diff
@@ -223,6 +233,15 @@ abstract class StandardCacheHitCopyingActor(val standardParams: StandardCacheHit
         case StillWaiting => stay() using Option(newData)
         case AllCommandsDone =>
           handleWhitelistingForSuccess(command)
+          // This is looking at the "before" data that should contain the last IoCommand we were waiting for.
+          data.commandsToWaitFor.flatten.headOption match {
+            case Some(command: IoCopyCommand) =>
+              if (command.isCopyWithinBucket) {
+                log.info(s"BT-322 $jobTag cache hit copy within the same bucket: ${command.source.pathAsString} -> ${command.destination.pathAsString}")
```

Comment:
ToL: I believe this satisfies BT-322.

I'm personally more curious to inspect events when files are copied between buckets. They could be in the same billing account (`gotc-1` to `gotc-2`) or from a shared dataset (`tcga-bams` to `my-custom-analysis`).

Maybe log both?